endpoints/calendar/index.js

```diff
@@ -1,11 +1,57 @@
 import app from '../../server'
 import fridagar from 'fridagar'
 import makeDebug from 'debug'
-import moment from 'moment'
-import { range } from 'lodash'
+import { range, isString } from 'lodash'
 
 const debug = makeDebug('endpoint:calendar')
 
+const canBeInt = intLike => {
+  const num = Number.parseInt(intLike, 10)
+  return !Number.isNaN(num)
+}
+
+export const normalizeParams = (year, month, day) => {
+  // If string parsing failed, reject the promise
+  if (isString(year) && !canBeInt(year)) return { error: 'Year must be a number' }
```

Comment:
I'm not sure. 
There might be a superior way to delegate validation to GraphQL. 
One way could be using https://github.com/graphql/express-graphql has a `validationRules` option key which could potentially use. If we were building a pure GraphQL server this would be perfect for validation.
But I'm not sure that fits our needs since for us that would mean duplicating error handling in two places.

1. in the GraphQL http server
2. in each of the REST endpoints

The current approach is:
- Handle all the validation and reject the Promise with errors in the common function shared between the GraphQL schema & the REST endpoints
- `catch` broken promises in the REST endpoints or the resolve functions of the GraphQL schema and decide how to respond to the client

which has the advantage that REST and GraphQL share as much logic as possible.

If anyone has a better idea I'm all for it.